.changeset/curly-wolves-promise.md

```diff
@@ -0,0 +1,9 @@
+---
+'@twilio-paste/design-tokens': patch
+'@twilio-paste/core': patch
+---
+
+[Design tokens] fix typo in gatsby tokens type declaration template and remove readonly declaration
+
+- Remove extra `{` character
+- Remove typing as a readonly object in the `gatsby.d.ts` temoplate
```

Comment:
thought: since the Version Packages PR hasn't merged yet, we can just piggy back on that changeset file and not have to announce to the world we broke something.  If possible I'd suggest updating that changeset file to something a little more external consumer friendly like:


"[Design tokens] Add a new generic export featuring additional metadata. This can be used in documentation or software tooling."

And then we can rename it to something like `generic.ts` instead of `gatsby.ts` and our friend with the VSCode extension can use it too.  What do you think?